File: csharp/rules/correctness/no-suppressed-shell-exit-codes.cs
public class ShellHelper
{
    public void Run()
    {
        // ruleid: csharp-no-suppressed-shell-exit-codes
        var cmd1 = "dotnet test || true";

        // ruleid: csharp-no-suppressed-shell-exit-codes
        var cmd2 = "cat file.txt 2>/dev/null";

        // ok: csharp-no-suppressed-shell-exit-codes
        var safeCmd = "dotnet test && dotnet build";
    }
}

File: python/rules/behavior-and-failure-handling/no-suppressed-shell-exit-codes.py
import subprocess

def run_commands():
    # ruleid: python-no-suppressed-shell-exit-codes
    cmd1 = "npm test || true"

    # ruleid: python-no-suppressed-shell-exit-codes
    cmd2 = "ls -la 2>/dev/null"

    # ruleid: python-no-suppressed-shell-exit-codes
    cmd3 = "cat log.txt > /dev/null 2>&1"

    # ok: python-no-suppressed-shell-exit-codes
    safe_cmd = "npm test && pytest"
